src/sv-pipeline/04_variant_resolution/scripts/merge_vcfs.py

```diff
@@ -35,6 +47,12 @@ def dedup_records(records):
     curr_record = records[0]
     for record in records[1:]:
         if records_match(curr_record, record):
+            # keep more informative ALT field
+            curr_alt = curr_record.alts[0]
+            new_alt = record.alts[0]
+            if (curr_alt.startswith('<') and curr_alt.endswith('>') and new_alt.startswith('<') and new_alt.endswith('>') and
```

Comment:
Yes, the reason I included the check for <> is because of BNDs like this, in which `[CHR15:101980213[N` is the ALT field: 
```
chr1    10457   gnomAD-SV_v3_mc5b21_manta_chr1_chr1_1   N       [CHR15:101980213[N      .       PASS    END=101980223;SVTYPE=BND;CHR2=chr15;STRANDS=--;SVLEN=-1;ALGORITHMS=manta;MEMBERS=manta___14697_cvd_hgv__45f8c7_1;EVIDENCE=SR    GT
```